app/src/main/java/cm/aptoide/pt/app/view/AppViewPresenter.java

```diff
@@ -408,9 +408,9 @@ private void handleClickOnSimilarApps() {
         .doOnNext(similarAppClickEvent -> {
           if (similarAppClickEvent.getSimilar()
               .isAd()) {
-            appViewAnalytics.sendSimilarAppsInteractEvent(similarAppClickEvent.getType());
-            appViewNavigator.navigateToAd(similarAppClickEvent.getSimilar()
-                .getAd());
+//            appViewAnalytics.sendSimilarAppsInteractEvent(similarAppClickEvent.getType());
+//            appViewNavigator.navigateToAd(similarAppClickEvent.getSimilar()
```

Comment:
commented code

Reply:
The only reason I'm not deleting this is so that in the next ticket (AB test), these events will not be forgotten and handled properly for both cases (either coming from Aptoide or AppNext)